netlify/functions/club/reviews.ts

```diff
@@ -64,4 +64,66 @@ router.put(
   },
 );
 
+router.get("/:workId/shared", async ({ clubId, params }, res) => {
```

Comment:
Should we do a validation token similar to the join? As it sits, if I knew the work id I could access anyones review. Super low stakes though, could tackle in the future

Reply:
True good idea or we could make the url use list item id or something so it's not consistent across clubs. idk i'll add it to the backlog